### PR TITLE
FixedSticky removed

### PR DESCRIPTION
**Pull request by hannahpurcell**
#### Summary of changes
**Asana Ticket:** [Tech Debt | Look into removing fixedsticky from front-end](https://app.asana.com/0/555089885850811/1173098961983546)

The fixedSticky mechanisms didn't seem to be hooked up or appear in the UI at all.  The only place where a fixedSticky style appeared in a UI was with the accordion rows seen on the bus fares page:  https://www.mbta.com/fares/bus-fares.  Removed the fixedSticky on the accordion did not impact rendered style.  Likewise, removing the references to the plugin made no impact.

### CODE REVIEW

File: apps/site/lib/site_web/views/schedule_view.ex [hunk 1]
```diff
@@ -54,6 +54,8 @@ defmodule SiteWeb.ScheduleView do
   @spec template_for_tab(String.t()) :: String.t()
   @doc "Returns the template for the selected tab."
   def template_for_tab(tab_name)
+  # Is trip-view ever rendered?
```
Comment (thecristen) [inline apps/site/lib/site_web/views/schedule_view.ex:57]: Oh hey! This is the Schedule tab that we are about to remove this March. So it is rendered, but not for much longer!

**Review by thecristen — COMMENTED** [review]
> Removed the fixedSticky on the accordion did not impact rendered style.

🎉 Did you check this on IE and on mobile too?

**Comment by hannahpurcell** [comment]
> > Removed the fixedSticky on the accordion did not impact rendered style.
> 
> 🎉 Did you check this on IE and on mobile too?

Oh, great question!  It wasn't working on IE in the first place!  And my updates made no change.  I might as well fix it in this branch, 'n take care of a little bug we didn't know existed!

**Comment by hannahpurcell** [comment]
> 🎉 Did you check this on IE and on mobile too?

Oh MY, I was totally off the trail on this one.  That's what the FixedSticky.js file was all about - checking to see if "sticky" is native to the browser, and if not, creating "stickyness" from scratch.  So this file and the other fixedSticky refs are exclusively needed for IE.  I don't know why it didn't click for me sooner

**Pull request closed without merging**